python/cxx/deck_keyword.cpp

```diff
@@ -148,7 +161,19 @@ void python::common::export_DeckKeyword(py::module& module) {
         .def( "__getitem__", getRecord, ref_internal)
         .def( "__len__", &DeckKeyword::size )
         .def_property_readonly("name", &DeckKeyword::name )
-        ;
+
+    .def(py::init([](const ParserKeyword& parser_keyword, py::array_t<int> py_data) {               
+            return DeckKeyword(parser_keyword, numpy_array_to_vector(py_data));
```

Comment:
There would be two copies per DeckKeyword initiation:
1) copy from numpy vector to a std::vector
2) push_back from std::vector to the std::vector inside the DeckItem. 
This could be circumvented with DeckKeyword taking int*/double* (the memory location of the numpy vector) instead of std::vector, but it is a bit messy solution.

Reply:
I agree on your point 2; both that it is possible, and that it is messy.